iosxr_iso2vbox.py

```diff
@@ -293,136 +513,149 @@ def cleanup_vms(name, box_name):
 
     # Add in embedded Vagrantfile
     vagrantfile_pathname = os.path.join(pathname, 'include', 'embedded_vagrantfile')
 
-    run('vagrant package --base %s --vagrantfile %s --output %s' % (vmname, vagrantfile_pathname, box_out))
-    verboseprint('Created: %s' % box_out)
+    run(['vagrant', 'package', '--base', vmname, '--vagrantfile', vagrantfile_pathname, '--output', box_out])
+    logger.info('Created: %s', box_out)
 
     # Create OVA
     if create_ova is True:
-        verboseprint('Creating OVA %s' % ova_out)
-        run('VBoxManage export %s --output %s' % (vmname, ova_out))
-        verboseprint('Created OVA %s' % ova_out)
-
-    # Run basic sanity tests
-    verboseprint('Running basic unit tests on VirtualBox...')
-    iosxr_test_path = os.path.join(pathname, 'iosxr_test.py')
-    cmdstring = "python %s %s" % (iosxr_test_path, box_out)
-    result = (subprocess.check_output(cmdstring, shell=True))
-    if result is False:
-        # Fail noisily
-        sys.exit('Failed basic test, box %s is not sane' % box_out)
-    else:
-        verboseprint('Passed basic test, box %s is sane' % box_out)
+        logger.info('Creating OVA %s', ova_out)
+        run(['VBoxManage', 'export', vmname, '--output', ova_out])
+        logger.debug('Created OVA %s', ova_out)
 
-    verboseprint('Single node use:')
-    verboseprint(" vagrant init 'IOS XRv'")
-    verboseprint(" vagrant box add --name 'IOS XRv' %s --force" % box_out)
-    verboseprint(' vagrant up')
+    if args.skip_test is False:
+        # Run basic sanity tests
+        logger.info('Running basic unit tests on Vagrant VirtualBox...')
 
-    verboseprint('Multinode use:')
-    verboseprint(" Copy './iosxrv-x64-vbox/vagrantfiles/simple-mixed-topo/Vagrantfile' to the directory running vagrant and do:")
-    verboseprint(" vagrant box add --name 'IOS XRv' %s --force" % box_out)
-    verboseprint(' vagrant up')
-    verboseprint(" Or: 'vagrant up rtr1', 'vagrant up rtr2'")
+        # Remove stale SSH entry
+        logger.debug('Removing stale SSH entries')
+        run(['ssh-keygen', '-R', '[localhost]:2222'])
+        run(['ssh-keygen', '-R', '[localhost]:2223'])
 
-    verboseprint('Note that both the XR Console and the XR linux shell username and password is vagrant/vagrant')
+        if args.verbose == logging.DEBUG:
+            verbose_str = '-v'
+        else:
+            # Shhhh...
+            verbose_str = ''
+        logger.debug("verbose string '%s'", verbose_str)
 
-    # Clean up existing vm's
-    cleanup_vms(vmname, vbox)
+        iosxr_test_path = os.path.join(pathname, 'iosxr_test.py')
+        cmd_string = "python %s %s %s" % (iosxr_test_path, box_out, verbose_str)
+        result = (subprocess.check_output(cmd_string, shell=True))
+
+        if result is False:
+            sys.exit('Failed basic test, box %s is not sane' % box_out)
+        else:
+            logger.info('Passed basic test, box %s is sane', box_out)
+
+    logger.debug('Single node use:')
```

Comment:
Feels a little odd to me that these instructions are debug logs - should this be print or logger.info instead?